databroker/src/broker.rs

```diff
@@ -153,17 +153,49 @@ pub struct QuerySubscription {
     permissions: Permissions,
 }
 
+#[cfg(not(feature="stats"))]
```

Comment:
If intended for main branch, should we possibly somewhere document what the feature "stats" is intended for and how it works.